packages/puppeteer-core/src/node/LaunchOptions.ts

```diff
@@ -25,6 +25,11 @@ import {Product} from '../common/Product.js';
 export interface BrowserLaunchArgumentOptions {
   /**
    * Whether to run the browser in headless mode.
+   *
+   * @remarks
+   * Deprecating `headless: true` if favor of `headless: 'new'`
```

Comment:
> `headless: true` will launch the new headless mode in the future equivalent to the current `headless: 'new'` option.

I think smth like this is better as we don't deprecate headless:true, just what is meant by it is changing.